packages/mobx-state-tree/src/core/mst-operations.ts

```diff
@@ -710,8 +710,8 @@ export function getEnv<T = any>(target: IAnyStateTreeNode): T {
     }
     const node = getStateTreeNode(target)
     const env = node.root._environment
-    if (!!!env) return EMPTY_OBJECT as T
-    return env
+
+    return env || fail(`Failed to find the environment of ${getStateTreeNode(target)}`)
```

Comment:
Maybe show the path in the error as well?

Reply:
I added the `path` in, do you think that's sufficient ?